src/components/common/ExchangeRate2.tsx

```diff
@@ -0,0 +1,33 @@
+// import { log10 } from 'biggystring'
+import { EdgeCurrencyWallet } from 'edge-core-js'
+import * as React from 'react'
+import { memo } from 'react'
+import { TextStyle } from 'react-native'
+
+import { useDisplayDenom } from '../../hooks/useDisplayDenom'
+import { getCurrencyCode } from '../../util/CurrencyInfoHelpers'
+import { FiatText } from '../text/FiatText'
+import { EdgeText } from '../themed/EdgeText'
+
+interface Props {
+  wallet: EdgeCurrencyWallet
+  tokenId?: string
+  style?: TextStyle
```

Comment:
We generally disallow passing style props to components. Text nodes do nest, you can always do `<EdgeText style={...}><ExchangeRate /></EdgeText>` to control font. sizing, colors, &c.

I am open to leaving the style if there is a good reason, though.

Reply:
This was really copy/paste from the original ExchangeRate component. Looks like it's only used for text size so I just moved the style into ExchangeRate2